challenges/stats_preview_card_component/index.html

```diff
@@ -25,39 +25,124 @@
       sizes="16x16"
       href="../images/favicon-16x16.png"
     />
+    <!-- Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Use one URL for multiple fonts to reduce the number of HTTP requests.

Reply:
I see the point, thanks

Reply:
> Use one URL for multiple fonts to reduce the number of HTTP requests.

I see know the point, thanks